cypress/e2e/with_mock_data/items.cy.ts

```diff
@@ -425,6 +423,186 @@ describe('Items', () => {
     cy.findByText('Zf7P8Qu8TD8c').should('exist');
   });
 
+  describe('Attachments', () => {
+    afterEach(() => {
+      cy.clearMocks();
+    });
+
+    it('uploads attachment', () => {
+      cy.findByText('5YUQDDjKpz2z').click();
+      cy.findByText(
+        'High-resolution cameras for beam characterization. 1'
+      ).should('exist');
+      cy.findByRole('button', {
+        name: 'items landing page actions menu',
+      }).click();
+      cy.findByText('Upload Attachments').click();
+
+      cy.findAllByText('Files cannot be larger than', { exact: false }).should(
+        'exist'
+      );
+      cy.get('.uppy-Dashboard-input').as('fileInput');
+
+      cy.get('@fileInput')
+        .first()
+        .selectFile(
+          [
+            'cypress/fixtures/documents/test1.txt',
+            'cypress/fixtures/documents/test2.txt',
+          ],
+          { force: true }
+        );
+      cy.startSnoopingBrowserMockedRequest();
+      cy.findByText('Upload 2 files').click();
+
+      cy.findBrowserMockedRequests({
+        method: 'POST',
+        url: '/attachments',
+      }).should(async (postRequests) => {
+        expect(postRequests.length).eq(2);
+        expect(JSON.stringify(await postRequests[0].json())).equal(
+          JSON.stringify({
+            entity_id: 'KvT2Ox7n',
+            file_name: 'test1.txt',
+          })
+        );
+        expect(JSON.stringify(await postRequests[1].json())).equal(
+          JSON.stringify({
+            entity_id: 'KvT2Ox7n',
+            file_name: 'test2.txt',
+          })
+        );
+      });
+
+      cy.findBrowserMockedRequests({
+        method: 'POST',
+        url: '/object-storage',
+      }).should(async (postRequests) => {
+        expect(postRequests.length).eq(2);
+      });
+      cy.findByText('Complete').should('exist');
+    });
+
+    it('errors when file is removed mid upload', () => {
+      cy.window().its('msw').should('not.equal', undefined);
+      cy.window().then((window) => {
+        const { worker, http } = window.msw;
+
+        worker.use(
+          http.post('/object-storage', async () => {
+            await delay(500);
+            return HttpResponse.json({}, { status: 200 });
```

Comment:
```suggestion
            return new HttpResponse(undefined, { status: 200 });
```

Reply:
what is the differences ?

Reply:
bar the empty dict that should be removed

Reply:
```
    console.log(
      JSON.stringify(new HttpResponse(undefined, { status: 200 })) ===
        JSON.stringify(HttpResponse.json(undefined, { status: 200 }))
    );
```

this returned back `true `. to keep it consistent with what we have done before I'm going to use `.json` method.